docs/hw/hookup.md

```diff
@@ -3,6 +3,39 @@
 The iRobot® Create® 3 has a USB-C®[^1] connector implementing a USB 2.0 host capable of 5 V @ 3 A continuous, which can be used to power and communicate with various downstream devices.
 Here are some hookup examples.
 
+## NXP NavQ+
+Since NavQ+ has USB-C® ports capable of being configured as a USB Ethernet Gadget, a cable with a USB-C® connector on both ends is all that is required.
+
+!!! note
+    - Unless set otherwise, the default USB-C® port for use as an ethernet gadget device is configured to only the centermost (USB 2) USB-C® as shown in the example pictures.
+
+The NavQ+ is pictured in the cargo bay using the 3D printed adapter plate and ontop of the faceplate using the 3D printed case.
```

Comment:
"3d-printed" should be hyphenated.

Reply:
"ontop" should be two words.

Reply:
Fixed in: 
[66daa95](https://github.com/iRobotEducation/create3_docs/pull/166/commits/66daa95db4b78119d7344dbae8c3cd17e8cc0ead)